projects/v2/src/environments/environment.prod.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
   production: true,
   tag: '',
-  googleAnalyticsId: 'UA-136932895-5'
+  googleAnalyticsId: 'UA-136932895-5',
+  apasctbApi: 'https://asctb-api.herokuapp.com/v2'
```

Comment:
What does the ap stand for? Either way let's call it asctbApiUrl.
Also, remove the /v2 and add it elsewhere where appropriate. We will likely be adding new /vX and other urls to that API soonish (ie for exporting bits to graph formats, etc).

Reply:
Done